pybird_dev/greenfunction.py

```diff
@@ -100,60 +209,71 @@ def G(self, a):
 
     # third order coefficients
     def IU1d(self, ai, a):
-        return self.fplus(ai)*self.mG1d(ai)*self.G1d(a,ai)*(self.D(ai)/self.D(a))**3 / self.C(a)
+        if self.MG: return (self.G1d(a,ai)*self.fplus(ai)*self.mG1d_MG(ai) + self.G2d(a,ai)*(1.5*self.Omega_m(ai))**2*(self.mu2(ai)*self.mG1d_MG(ai) + 0.5*self.mu22(ai)*1.5*self.Omega_m(ai))/self.fplus(ai))*(self.D(ai)/self.D(a))**3 / self.C(a)
```

Comment:
In the following, why the _MG subscript to the various mG1d etc. functions? I think they are defined without it

Reply:
You're correct, these _MG subscripts remained from a previous implementation. I have removed these in commit #5860bc5.